### PR TITLE
LPD-35158

### PR DESCRIPTION
**Pull request by cdbm**

### CODE REVIEW

**Comment by cdbm** [comment]
ci:test:relevant

File: modules/dxp/apps/scim/scim-configuration-web-test/src/testIntegration/java/com/liferay/scim/configuration/web/internal/portlet/action/test/SaveScimConfigurationMVCActionCommandTest.java [hunk 1]
```diff
@@ -142,9 +136,9 @@ public void testOAuth2TokenIsIssuedToClientCredentialUser()
 
 		Assert.assertEquals(
 			adminUser.getUserId(), oAuth2Authorization.getUserId());
-	}
 
-	private static String _pid;
+		ConfigurationTestUtil.deleteConfiguration(pid);
+	}
```
Comment (stian-sigvartsen) [inline modules/dxp/apps/scim/scim-configuration-web-test/src/testIntegration/java/com/liferay/scim/configuration/web/internal/portlet/action/test/SaveScimConfigurationMVCActionCommandTest.java:141]: HI @cdbm . You should ensure this gets executed by using a try + finally block

**Comment by cdbm** [comment]
ci:test:sf

**Comment by stian-sigvartsen** [comment]
I just added a slight tweak and manually forwarded to brianchandotcom/liferay-portal/pull/154371

**Comment by stian-sigvartsen** [comment]
There was a SF error with my change. Sent again to brianchandotcom/liferay-portal/pull/154372

**Pull request closed without merging**